main.tf

```diff
@@ -122,30 +113,20 @@ resource "azurerm_key_vault_access_policy" "vault_init" {
 }
 
 module "vault_identity" {
-  source = "/Users/tmiller/gitlab-repos/tfe/terraform-azurerm-kubernetes/identity"
+  source = "/Users/tmiller/gitlab-repos/tfe/terraform-azurerm-kubernetes/aad-pod-identity/identity"
```

Comment:
Does this need to change?

Reply:
yep, was a remnant from development, thanks